src/Akeneo/Bundle/ElasticSearchBundle/AkeneoElasticSearchBundle.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Benoit made a remark in one of my PR that ES stands for Elasticsearch not Elastic Search so maybe change the bundle name for `AkeneoElasticSearchBundle`. Now, You choose.